packages/simcore-sdk/tests/integration/test_node_ports_common_filemanager.py

```diff
@@ -21,17 +27,45 @@
 pytest_simcore_ops_services_selection = ["minio", "adminer"]
 
 
-async def test_valid_upload_download(
-    tmpdir: Path,
+@pytest.fixture
+def node_ports_config(
     bucket: str,
     filemanager_cfg: None,
+    cleanup_file_meta_data: None,
+):
+    ...
+
+
+@pytest.fixture(params=[True, False], ids=["with RClone", "without RClone"])
+def optional_r_clone(
+    r_clone_settings: RCloneSettings, request: pytest.FixtureRequest
+) -> Optional[RCloneSettings]:
+    return r_clone_settings if request.param else None  # type: ignore
+
+
+def _file_size(size_str: str, **pytest_params):
+    return pytest.param(parse_obj_as(ByteSize, size_str), id=size_str, **pytest_params)
+
+
+@pytest.mark.parametrize(
+    "file_size",
+    [
+        _file_size("10Mib"),
+        _file_size("1003Mib"),
+        _file_size("7Gib", marks=pytest.mark.heavy_load),

```

Comment:
Question: will testing with a 7GB file take a really long time? I do like having it though.

Reply:
it's ok on my machine, not so much on the CI where the Rclone took 6 minutes...